.github/actions/setup-sentry/action.yml

```diff
@@ -56,6 +56,7 @@ runs:
         [ "$GITHUB_REF" = "refs/heads/master" ] && echo "PYTEST_SENTRY_ALWAYS_REPORT=1" >> $GITHUB_ENV || true
 
         ### services configuration ###
+        echo "BIGTABLE_EMULATOR_HOST=localhost:8086" >> $GITHUB_ENV
```

Comment:
This is going to be used for roughly a half dozen tests, but this container takes negligible resources and sets up quickly. It doesn't seem to me that this would merit a separate Bigtable-specific matrix step unless there is some philosophical reason to split it out.